src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -5,14 +5,113 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import codeu.model.data.User;
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.util.UUID;
+import java.util.List;
+import codeu.model.store.basic.UserStore;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+import codeu.model.data.User;
+import codeu.model.data.Message;
 
-
```

Comment:
Would sort your imports alphabetically. And also leave the blank line before the javadoc at the top of the class.

Reply:
Hi Julie, do you mean my imports at the top of the page? Also, what is the PR description?

Reply:
Yes I mean import codeu.model.data.User; etc,

PR description is your comment above (starts with "pull request - "). Just helps your team understand what exactly you're doing in this code.